app/experimenter/experiments/bugzilla.py

```diff
@@ -89,6 +98,24 @@ def create_experiment_bug(experiment):
         response_data = make_bugzilla_call(
             settings.BUGZILLA_CREATE_URL, bug_data
         )
+
+    # Firefox Version given might not be an available
+    # bugzilla tracking parameter, so remove and retry
+    err_code = response_data.get("code", None) == INVALID_PARAMETER_ERROR_CODE
+    if err_code:
+        tracking_msg = "cf_tracking_firefox" in response_data.get(
+            "message", None
+        )
+    if err_code and tracking_msg:
+        bug_data = bug_data.copy()
+        cf_tracking = "cf_tracking_firefox{}".format(
```

Comment:
Probably just move this into a property method right on experiment:

    experiment.bugzilla_tracking_key

so that you can call it from the format_bug_data function and here without repeating it.